src/api-client/server/index.ts

```diff
@@ -144,7 +144,9 @@ export default {
       return uri;
     }
 
-    return `${process.env.VUE_APP_BACKEND_URL}/media/${md5(uri)}`;
+    return `${process.env.VUE_APP_BACKEND_URL}${
+      process.env.VUE_APP_BACKEND_URL?.endsWith("/") ? "" : "/"
```

Comment:
I would just remove the slash and don't put in an additional check. Maybe `VUE_APP_BACKEND_URL` is the wrong name and `VUE_APP_BACKEND_HOST` would be a better choice.